packages/components/src/Contributors/ContributorCount.tsx

```diff
@@ -32,7 +32,7 @@ export class ContributorCount extends React.Component<ContributorsCountProps> {
           {this.props.displayedContributors.slice(0, 3).map((contributor: any, i: number) => {
             return (
               <ContributorCountAvatar key={i}>
-                {contributor.payerChannel.tiny72AvatarDataUrl ? (
+                {contributor.payerChannel && contributor.payerChannel.tiny72AvatarDataUrl ? (
```

Comment:
might be a conflict on this line cause I already fixed this in another PR, didn't I?

Reply:
You sure did - my commit disappeared upon rebasing.